sangwon2/models/postDao.js

```diff
@@ -0,0 +1,103 @@
+const appDataSource = require('./dataSource');
+
+const createPosts = async (userId, postImage, postText) => {
+  try {
+    return await appDataSource.query(
+      `INSERT INTO posts (
+          user_id,
+          post_image,
+          post_paragraph
+      ) VALUES (
+          ?,
+          ?,
+          ?
+      )`,
+      [userId, postImage, postText]
+    );
+  } catch (err) {
+        const error = new Error("INVALID_DATA_INPUT");
+        error.statusCode = 500;
+    throw error;
+  }
+};
+
+const showAllposts = async (userId, postImage, postText) => {
+    try {
+      return await appDataSource.query(`
+        SELECT 
+            id,
+            user_id,
+            post_image,
+            post_paragraph
+        FROM posts
+      `);
+    } 
+    catch (err) {
+      const error = new Error("INVALID_DATA");
+      error.statusCode = 500;
+      throw error;
+    }
+  };
+
+const getPostById = async (userId) => {
```

Comment:
함수 이름이 post의 상세 내용을 조회하는 역할 할 것 같습니다. 다음과 같이 수정하면 좋을 것 같습니다.
```suggestion
const getUserPosts = async (userId) => {
```